var/spack/repos/builtin/packages/charm/package.py

```diff
@@ -115,16 +161,11 @@ def install(self, spec, prefix):
         # We assume that Spack's compiler wrappers make this work. If
         # not, then we need to query the compiler vendor from Spack
         # here.
-        compiler = os.path.basename(self.compiler.cc)
-
-        options = [compiler]
-        if compiler == 'icc':
-            options.append('ifort')
-
-        options.extend([
-            "--with-production",   # Note: turn this into a variant
-            "-j%d" % make_jobs,
```

Comment:
Just noticed this. Why did you drop parallel support?

Reply:
I saw that it can be done in spack already by doing spack install -j #numjobs, so I removed it

Reply:
So when you do `spack install -j #numjobs`, Spack sets the `make_jobs` variable to this value. If you run `make()` [or one of these other functions](https://github.com/spack/spack/blob/develop/lib/spack/spack/build_environment.py#L362,#L365), Spack automatically adds `-j makejobs` to the call. However, in this case, we don't seem to be using `make()`, so it is still needed.